docs/kubeadm.md

```diff
@@ -0,0 +1,210 @@
+
+You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
+
+### Installing a sample application
+
+As an example, you will now install a sample microservices application, a socks shop, to put your cluster through its paces.
+To learn more about the sample microservices app, see the [GitHub README](https://github.com/microservices-demo/microservices-demo).
+
+Here you will install the NodePort version of the Socks Shop, which doesn't depend on Load Balancer integration, since our cluster doesn't have that:
+
+    # kubectl apply -f https://raw.githubusercontent.com/lukemarsden/microservices-demo/master/deploy/kubernetes/definitions/wholeWeaveDemo-NodePort.yaml
+
+You can then find out the port that the [NodePort feature of services](/docs/user-guide/services/) allocated for the front-end service by running:
+
+    # kubectl describe svc front-end
+    ... XXX ...
+
+Then go to the IP address of your cluster's master node in your browser, and specify the given port.
+So for example, `http://<master_ip>:<port>`.
+
+If there is a firewall, make sure it exposes the port to the internet before you try to access it.
+
+### Explore other add-ons
+
+See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+
+
+## What's next
+
+* Learn more about [Kubernetes concepts and kubectl in Kubernetes 101](/docs/user-guide/walkthrough/).
+* Install Kubernetes with [a cloud provider configurations](/docs/getting-started-guides/) to add Load Balancer and Persistent Volume support.
+
+
+## Cleanup
+
+* To uninstall the socks shop, run `kubectl delete -f https://raw.githubusercontent.com/lukemarsden/microservices-demo/master/deploy/kubernetes/definitions/wholeWeaveDemo-NodePort.yaml`.
+* To uninstall Kubernetes, simply delete the machines you created for this tutorial.
+  Or alternatively, uninstall the `kubelet`, `kubeadm` and `kubectl` packages and then manually delete all the Docker container that were created by this process.
+
+## How to provide feedback
+
+* Slack Channel: [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
+* Mailing List: [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+* [GitHub Issues](https://github.com/kubernetes/kubernetes/issues): please tag `kubeadm` issues with `@kubernetes/sig-cluster-lifecycle`
+
+## Limitations
+
+1. The cluster we create here won't have cloud-provider integrations, so for example won't work with (for example) [Load Balancers](/docs/user-guide/load-balancer/) (LBs) or [Persistent Volumes](/docs/user-guide/persistent-volumes/walkthrough/) (PVs).
+   
+   Instead we will use the [NodePort feature of services](/docs/user-guide/services/) to demonstrate exposing the sample application on the internet.
```

Comment:
This line is in the future tense, but the user has already _used_ the NodePort in the past :)